ansible/roles/sunbird-system-init/templates/system_init_commands.j2

```diff
@@ -0,0 +1,336 @@
+                -H 'authorization: Bearer '"$ekstep_api_key" \
+                -H 'cache-control: no-cache' \
+                -H 'content-type: application/json' \
+                -d '{
+                        "request": {
+                                "channel": {
+                                        "name":"'"$sunbird_custodian_tenant_channel"'",
+                                        "description":"'"$sunbird_custodian_tenant_description"'",
+                                        "code":"'"$sunbird_custodian_tenant_id"'"
+                                }
+                        }
+                    }')
+}
+
+registerHashTag() {
+        register_hashtag_response_code=$(curl -o /dev/null -s -w "%{http_code}\n" \
+                -X POST $ekstep_proxy_base_url/api/data/v3/tag/register/$sunbird_custodian_tenant_id \
+                -H 'authorization: Bearer '"$ekstep_api_key" \
+                -H 'cache-control: no-cache' \
+                -H 'content-type: application/json' \
+                -d '{}')
+}
+
+createOrgInCassandra() {
+        sunbird_cassandra_org_insert_query="insert into sunbird.organisation \
+        (orgname,description,channel,slug,id,hashtagid, \
+        rootorgid,createddate,createdby,isrootorg,isdefault, status) values \
+        ('""$sunbird_custodian_tenant_name""', '""$sunbird_custodian_tenant_description""', \
+        '""$sunbird_custodian_tenant_channel""', '""$sunbird_custodian_tenant_slug""', \
+        '""$sunbird_custodian_tenant_id""', '""$sunbird_custodian_tenant_id""', \
+        '""$sunbird_custodian_tenant_id""', '""$currenttime""', 'system', true, true, 1);"
+
+        cqlsh -e "$sunbird_cassandra_org_insert_query"
+}
+
+createOrgInES() {
+        es_host="$(echo "$sunbird_es_host" | cut -d ',' -f1)"
+        es_create_org_response=$(curl -X PUT $es_host:9200/searchindex/org/$sunbird_custodian_tenant_id \
+                -H 'cache-control: no-cache' \
+                -H 'content-type: application/json' \
+                -d '{
+                        "orgName":"'"$sunbird_custodian_tenant_name"'",
+                        "description":"'"$sunbird_custodian_tenant_description"'",
+                        "channel":"'"$sunbird_custodian_tenant_channel"'",
+                        "slug":"'"$sunbird_custodian_tenant_slug"'",
+                        "id":"'"$sunbird_custodian_tenant_id"'",
+                        "identifier":"'"$sunbird_custodian_tenant_id"'",
+                        "hashTagId":"'"$sunbird_custodian_tenant_id"'",
+                        "rootOrgId":"'"$sunbird_custodian_tenant_id"'",
+                        "createdDate":"'"$currenttime"'",
+                        "createdBy":"system",
+                        "isRootOrg":true,
+                        "isDefault":true,
+                        "status":1
+                    }')
+}
+
+fetchOrgFromES() {
+        es_host="$(echo "$sunbird_es_host" | cut -d ',' -f1)"
```

Comment:
Getting following error in fetchOrgFromES.

curl: (3) [globbing] nested brace in column 2

Reply:
Above error occurs when:
sunbird_es_host="{{sunbird_es_host}}"

Reply:
@bvinayakumar When we run this script using ansible playbook, `{{sunbird_es_host}}` will be replaced with the configured value(ip address/dns name). If the variable is not configured, error will be thrown.
I suppose the script is being tested standalone. So I suggest to replace `{{sunbird_es_host}}` with valid values(ip address/dns name) and try ES related methods. For failure test cases, the variable may be given invalid values or left blank.